### DISCUSSION TITLE
Scrollbar themed > how to deactivate it?

### DISCUSSION THREAD

**Discussion by skeud** [adityatelange/hugo-PaperMod#904] [Q&A]
Hi there,

I'm getting my hands into Hugo and your great theme.
May you explain the way to deactivate the themed scrollbar (fat and grey) within the config.yml please?

Thanks!

**Comment by adityatelange** [comment]
No way yet you can do it yourself if you know css and by using extended css options to override this

**Comment by skeud** [comment]
Thank you!
